.experimenter.yaml

```diff
@@ -58,6 +58,14 @@ nimbus-validation:
     settings-title:
       type: string
       description: The title of displayed in the Settings screen and app menu.
+onboardingcreen:
```

Comment:
I am not the most familiar person with the experimenter.yaml and nimbus.fml.yml. So, you'll want to get another review from someone who is more familiar with the two.